Makefile

```
@@ -17,3 +17,11 @@ download-oas:
 generate-sdk:
 	@$(SCRIPTS_BASE)/generate-sdk/generate-sdk.sh "$(GIT_HOST)" "$(GIT_USER_ID)" "$(GIT_REPO_ID)" "$(SDK_REPO_URL)" "$(LANGUAGE)" "$(SDK_BRANCH)"
 
+generate-go-sdk: LANGUAGE=go
```

```suggestion
generate-go-sdk: 
    @$(SCRIPTS_BASE)/generate-sdk/generate-sdk.sh "$(GIT_HOST)" "$(GIT_USER_ID)" "$(GIT_REPO_ID)" "$(SDK_REPO_URL)" "GO" "$(SDK_BRANCH)"
```

If you don't do it like this, we won't be able to use the `generate-go-sdk` (and the other new make targets) with other branches, git repos, ...